src/main/scala/cromwell/Main.scala

```diff
@@ -19,68 +20,34 @@ object Main extends App {
 
   getAction(args.headOption) match {
     case Some(x) if x == Actions.parse => parseIt(args.tail)
+    case Some(x) if x == Actions.validate => validateIt(args.tail)
     case Some(x) if x == Actions.run => runIt(args.tail)
     case Some(x) if x == Actions.server => CromwellServer // Mention it so it gets instantiated
     case _ => usageAndExit()
   }
 
-  def printProcessedBinding(binding: WdlBinding): Unit = {
-    val workflow =
-      s"""
-         |Workflow FQN: ${binding.workflow.fullyQualifiedName}
-         |Workflow Inputs: ${binding.workflow.inputs}
-         |Workflow Outputs: ${binding.workflow.outputs}
-     """.stripMargin
-    println(workflow.trim + "\n")
-
-    val params = Map(
-      "in_file" -> WdlFile(Paths.get("/usr/share/dict/words")),
-      "pattern" -> WdlString("^...$")
-    )
-
-    binding.workflow.calls.foreach { call =>
-      val callString =
-        s"""
-           |Call: $call
-           |FQN: ${call.fullyQualifiedName}
-           |Unsatisfied Inputs: ${call.unsatisfiedInputs}
-           |Parent: ${call.parent}
-         """.stripMargin
-      println(callString + "\n")
-    }
-
-    binding.tasks.foreach { task =>
-      val taskString =
-        s"""
-           |Task: ${task.name}
-           |Abstract Command: ${task.command}
-           |Inputs: ${task.command.inputs}
-           |Instantiated: ${task.command.instantiate(params)}
-         """.stripMargin
-      println(taskString + "\n")
-    }
+  def parseIt(args: Array[String]): Unit = {
+    if (args.isEmpty) usageAndExit()
+    else println(WdlBinding.getAst(new File(args(0))).toPrettyString)
   }
 
-  def parseIt(args: Array[String]): Unit = {
+  def validateIt(args: Array[String]): Unit = {
+    try {
+      val binding = WdlBinding.process(new File(args(0)))
+      println("WDL File is valid")
+    } catch {
+      case e:SyntaxError => println(e)
+    }
```

Comment:
They're *kinda* overlapping.  Parsing won't do higher-level syntax checks and it'll specifically print out an AST (mostly useful for developers).  Whereas validate is meant more for users and it will do the full gamut of syntax checks